packages/app/src/containers/Workspace.tsx

```diff
@@ -47,6 +51,7 @@ const MainView = styled.main(({ theme }) => ({
 const ContentArea = styled.div(({ theme }) => ({
   flex: '1',
   display: 'flex',
+  overflow: 'hidden'
```

Comment:
I think we can add a wrapper to the element with `display: grid` that has `overflowY: auto` to fix this. We do that in `GridView.tsx` so that other components don't need to think about it

Reply:
I changed to use the container you suggested. It's all in `GridView.tsx` now.